win32ss/user/ntuser/window.c

```diff
@@ -3689,39 +3720,39 @@ co_IntSetWindowLong(HWND hWnd, DWORD Index, LONG NewValue, BOOL Ansi, BOOL bAlte
                 co_IntSendMessage(hWnd, WM_STYLECHANGED, GWL_STYLE, (LPARAM) &Style);
             break;
 
-         case GWL_WNDPROC:
+         case GWL_WNDPROC: // LONG_PTR
```

Comment:
You should switch (at least) the LONG_PTR ones over to `GWLP_xxx`. In PSDK headers, GWL_WNDPROC & friends are not defined for x64.

Reply:
Many of these need cleaning up too if you're interested ;)
https://git.reactos.org/?p=reactos.git&a=search&h=HEAD&st=grep&s=GWL_HINSTANCE
https://git.reactos.org/?p=reactos.git&a=search&h=HEAD&st=grep&s=GWL_WNDPROC
https://git.reactos.org/?p=reactos.git&a=search&h=HEAD&st=grep&s=GWL_USERDATA

Or we could add this to Jira as a good newbie task.